jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/ConnectionMetaData.java

```diff
@@ -52,7 +52,9 @@ default boolean isSecure()
 
     /**
      * @return whether the functionality of pushing resources is supported
+     * @deprecated in favour of 103 Early Hints
      */
+    @Deprecated(since = "12.0.1")
```

Comment:
Remove this, as I understand from your comment was slipped in unintentionally?

Reply:
It is also the wrong version string